File: Ruby/1047.rb
hora_inicial, minuto_inicial, hora_final, minuto_final = gets.chomp.split().map{|x|x.to_i}
tempo = ( (hora_final * 60) + minuto_final ) - ( (hora_inicial * 60) + minuto_inicial )
if (tempo <= 0) then
	tempo = tempo + (24 * 60)
end
horas = tempo / 60
minutos = tempo % 60
puts ("O JOGO DUROU %d HORA(S) E %d MINUTO(S)" % [horas, minutos])

File: Python/1047.py
hora_inicial, minuto_inicial, hora_final, minuto_final = list(map(int, input().split()))
tempo = ( (hora_final * 60) + minuto_final ) - ( (hora_inicial * 60) + minuto_inicial )
if (tempo <= 0):
	tempo += 24 * 60
horas = tempo // 60
minutos = tempo % 60
print(f"O JOGO DUROU {horas} HORA(S) E {minutos} MINUTO(S)")
